tasks/fix-cat2.yml

```diff
@@ -40,54 +36,75 @@
       - dod_logon_banner
 
 - name: "MEDIUM | RHEL-07-010060 | PATCH | The Red Hat Enterprise Linux operating system must enable a user session lock until that user re-establishes access using established identification and authentication procedures."
-  command: "true"
-  changed_when: no
+  blockinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/00-screensaver
+      block: |
+        [org/gnome/desktop/screensaver]
+        lock-enabled=true
+  notify: dconfupdate
   when:
       - rhel7stig_gui
       - rhel_07_010060
   tags:
       - RHEL-07-010060
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-010061 | PATCH | The Red Hat Enterprise Linux operating system must uniquely identify and must authenticate users using multifactor authentication via a graphical user logon."
-  command: "true"
-  changed_when: no
+  lineinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/00-defaults
+      insertafter: '^#[org/gnome/login-screen]'
+      regexp: '^#?enable-smartcard-authentication'
+      line: enable-smartcard-authentication=true
+  notify: dconfupdate
   when:
       - rhel7stig_gui
       - rhel_07_010061
   tags:
       - RHEL-07-010061
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-010070 | PATCH | The Red Hat Enterprise Linux operating system must initiate a screensaver after a 15-minute period of inactivity for graphical user interfaces."
-  command: "true"
-  changed_when: no
+  lineinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/00-screensaver
+      insertafter: '^#[org/gnome/desktop/screensaver]'
+      regexp: '^#?idle-delay'
+      line: idle-delay=uint32 900
+  register: rhel_07_010070linetask
+  notify: dconfupdate
   when:
       - rhel7stig_gui
       - rhel_07_010070
   tags:
       - RHEL-07-010070
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-010081 | PATCH | The Red Hat Enterprise Linux operating system must prevent a user from overriding the screensaver lock-delay setting for the graphical user interface."
-  command: "true"
-  changed_when: no
+  lineinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/locks/session
+      regexp: '^#?lock-delay'
+      line: /org/gnome/desktop/screensaver/lock-delay
+  register: rhel_07_010081linetask
+  notify: dconfupdate
   when:
       - rhel7stig_gui
       - rhel_07_010081
   tags:
       - RHEL-07-010081
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-010082 | PATCH | The Red Hat Enterprise Linux operating system must prevent a user from overriding the session idle-delay setting for the graphical user interface."
-  command: "true"
-  changed_when: no
+  lineinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/locks/session
+      regexp: '^#?idle-delay'
```

Comment:
Which line are you trying to replace?  the `regexp` doesn't match the `line`.

Reply:
This must have been committed pre-cleanup. a recommit will change that on the next round.